src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/misc/StrangeNetherGoo.java

```diff
@@ -56,4 +66,26 @@ public ItemUseHandler getItemHandler() {
         };
     }
 
+    private EntityInteractHandler onRightClickEntity() {
+        return (e, item, hand) -> {
+            if ((e.getRightClicked() instanceof Sheep) && (e.getRightClicked() instanceof LivingEntity)) {
```

Comment:
This is redundant, a Sheep is an extension of a LivingEntity, therefore if something is a Sheep, it's also a LivingEntity.

```suggestion
            if (e.getRightClicked() instanceof Sheep) {
```

Reply:
Was right there `Methods inherited from interface org.bukkit.entity.LivingEntity`. I need to learn how to use Javadocs :D